InAppUtils/InAppUtils.m

```diff
@@ -7,13 +7,12 @@
 @implementation InAppUtils
 {
     NSArray *products;
-    NSMutableDictionary *_callbacks;
+    bool hasListeners;
 }
 
 - (instancetype)init
 {
     if ((self = [super init])) {
-        _callbacks = [[NSMutableDictionary alloc] init];
         [[SKPaymentQueue defaultQueue] addTransactionObserver:self];
```

Comment:
Wanneer willen we addTransactionObserver  aanroepen? willen we al naar de queue luisteren als we nog geen JS event listeners hebben?

Reply:
Als we niet luisteren krijgen we niet meer informatie toch? Of wachten ze met sturen van events totdat wij een observer toevoegen?

Reply:
Your application should add an observer to the payment queue during application initialization. If there are no observers attached to the queue, the payment queue does not synchronize its list of pending transactions with the Apple App Store, because there is no observer to respond to updated transactions.

Seems okay